simulator-core/src/test/java/com/evbox/everon/ocpp/mock/ocpp/RequestResponseSynchronizer.java

```diff
@@ -0,0 +1,67 @@
+package com.evbox.everon.ocpp.mock.ocpp;
```

Comment:
I'd prefer to move all server-related classes (including this one) to separate package like com.evbox.everon.ocpp.mock.ocpp.csms
Right now it's hard to understand which parts are used to emulate server behavior and which aren't

Reply:
will change the package name to csms